tests/network/localnet/conftest.py

```diff
@@ -0,0 +1,110 @@
+from libs.net import netattachdef
+from libs.net.traffic_generator import Client, Server
+from libs.net.vmspec import IP_ADDRESS, lookup_iface_status
+from libs.vm.vm import BaseVirtualMachine
+from tests.network.localnet.liblocalnet import NETWORK_NAME, localnet_nad, localnet_vm
+from utilities.constants import (
+    WORKER_NODE_LABEL_KEY,
+)
+from utilities.infra import create_ns
+
+_IPERF_SERVER_PORT = 5201
+
+
+@pytest.fixture(scope="module")
+def nncp_ovn_bridge_mapping() -> Iterator[libnncp.NodeNetworkConfigurationPolicy]:
+    desired_state = libnncp.DesiredState(
+        ovn=libnncp.OVN([
+            libnncp.BridgeMappings(
+                localnet=NETWORK_NAME,
+                bridge=libnncp.DEFAULT_OVN_EXTERNAL_BRIDGE,
+                state=libnncp.BridgeMappings.State.PRESENT.value,
+            )
+        ])
+    )
+
+    with libnncp.NodeNetworkConfigurationPolicy(
+        name="test-localnet-nncp",
+        desired_state=desired_state,
+        node_selector={WORKER_NODE_LABEL_KEY: ""},
+    ) as nncp:
+        nncp.wait_for_status_success()
+        yield nncp
+
+
+@pytest.fixture(scope="module")
+def vlan_id(vlan_index_number: Iterator[int]) -> int:
+    return next(vlan_index_number)
+
+
+@pytest.fixture(scope="module")
+def namespace_localnet_1() -> Iterator[Namespace]:
+    yield from create_ns(name="test-localnet-ns1")  # type: ignore
+
+
+@pytest.fixture(scope="module")
+def namespace_localnet_2() -> Iterator[Namespace]:
+    yield from create_ns(name="test-localnet-ns2")  # type: ignore
+
+
+@pytest.fixture(scope="module")
+def nad_localnet_1(namespace_localnet_1: Namespace, vlan_id: int) -> Iterator[netattachdef.NetworkAttachmentDefinition]:
+    with localnet_nad(namespace=namespace_localnet_1.name, name="test-localnet-nad1", vlan_id=vlan_id) as nad:
+        yield nad
+
+
+@pytest.fixture(scope="module")
+def nad_localnet_2(namespace_localnet_2: Namespace, vlan_id: int) -> Iterator[netattachdef.NetworkAttachmentDefinition]:
+    with localnet_nad(namespace=namespace_localnet_2.name, name="test-localnet-nad2", vlan_id=vlan_id) as nad:
+        yield nad
+
+
+@pytest.fixture(scope="module")
+def localnet_vms(
+    nad_localnet_1: netattachdef.NetworkAttachmentDefinition,
+    nad_localnet_2: netattachdef.NetworkAttachmentDefinition,
+) -> Iterator[tuple[BaseVirtualMachine, BaseVirtualMachine]]:
+    with (
+        localnet_vm(
+            namespace=nad_localnet_1.namespace, name="test-vm1", network=nad_localnet_1.name, cidr="10.0.0.1/24"
+        ) as vm1,
+        localnet_vm(
+            namespace=nad_localnet_2.namespace, name="test-vm2", network=nad_localnet_2.name, cidr="10.0.0.2/24"
+        ) as vm2,
+    ):
+        vms = (vm1, vm2)
+        for vm in vms:
+            vm.start()
+        for vm in vms:
+            vm.wait_for_ready_status(status=True)
+            vm.vmi.wait_for_condition(
+                condition=Resource.Condition.Type.AGENT_CONNECTED, status=Resource.Condition.Status.TRUE
```

Comment:
```suggestion
                condition=vm.Condition.Type.AGENT_CONNECTED, status=vm.Condition.Status.TRUE
```
(removes extra import that is not needed)

Reply:
Done.